src/query/storages/fuse/src/io/read/bloom_index_reader.rs

```diff
@@ -275,9 +276,10 @@ mod util_v1 {
                 Ok((bytes, idx))
             }
         } else {
-            Err(ErrorCode::Internal(format!(
-                "failed to find bloom index column. no such column {col_name}"
-            )))
+            warn!(
+                "failed to find bloom index column. no such column {col_name}, return empty"
+            );
+            Ok((Arc::new(vec![0u8]), 0))
```

Comment:
This modify is useless. I think we can catch read_filter err and return Ok(). That's can cover the table has data filed but not has bloom filter index.

After this pr, just optimize table or insert a value and then re-run analyze table, the fuse_static will correct.

Reply:
why not be `vec![]` ?